protos/offboard/offboard.proto

```diff
@@ -63,6 +69,18 @@ message Attitude {
   float thrust_value = 4;
 }
 
+message ActuatorControl {
+  enum ActuatorGroup {
+    FLIGHT_CONTROL = 0;
+    FLIGHT_CONTROL_VTOL = 1;
+    GIMBAL = 2;
+    MANUAL_PASSTHROUGH = 3;
+  }
+
+  ActuatorGroup actuator_group = 1;
+  repeated float actuator_values = 2 [packed=true];
```

Comment:
I believe `[packed=true]` is not necessary with [proto3](https://developers.google.com/protocol-buffers/docs/proto3):

> In proto3, repeated fields of scalar numeric types use packed encoding by default.